ush/ioda/bufr2ioda/bufr2ioda_adpupa_prepbufr.py

```diff
@@ -0,0 +1,471 @@
+# (C) Copyright 2023 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+import sys
+import argparse
+import numpy as np
+import numpy.ma as ma
+import calendar
+import json
+import time
+import math
+import datetime
+import os
+from datetime import datetime
+from pyioda import ioda_obs_space as ioda_ospace
+from wxflow import Logger
+from pyiodaconv import bufr
+from collections import namedtuple
+
+#|------------------------------------------------------------------------------|
+#|            gdas prepbufr file                                                |                    
+#|------------------------------------------------------------------------------|
+#| MNEMONIC | NUMBER | DESCRIPTION                                              |
+#|----------|--------|----------------------------------------------------------|
+#|          |        |                                                          |
+#| ADPUPA   | A48102 | UPPER-AIR (RAOB, PIBAL, RECCO, DROPS) REPORTS            |
+#|------------------------------------------------------------------------------|
+
+# Define and initialize  global variables
+global float32_fill_value
```

Comment:
Do we still need lines 32-38? I thought Ron's code made it so we don't need to declare these anymore.

Reply:
@nicholasesposito thanks, I commented these lines. Once reviewers agree, I will remove all such lines from python script.

Reply:
@nicholasesposito removed extra lines.

Reply:
@nicholasesposito done.